src/compiler/constraints.jl

```diff
@@ -160,6 +160,69 @@ function constraint(
     return g^2
 end
 
+@doc raw"""
+    constraint(
+        model::Virtual.Model{T}, 
+        f::SAF{T}, 
+        s::GT{T}, 
+        ::AbstractArchitecture
+    ) where {T}
+
+Turns constraints of the form
+
+```math
+\begin{array}{rl}
+\text{s.t} & \mathbf{a}'\mathbf{x} - b \ge 0
+\end{array}
+```
+
+into 
+
+```math
+\left\Vert(\mathbf{x})\right\Vert_{\left\lbrace{0}\right\rbrace} = (\mathbf{a}'\mathbf{x} - b - z)^{2}
+
+```
+
+by adding a slack variable ``z``.
+"""
+function constraint(
+    model::Virtual.Model{T},
+    f::SAF{T},
+    s::GT{T},
+    arch::AbstractArchitecture,
+) where {T}
+    # Scalar Affine Inequality Constraint: g(x) = a'x - b ≥ 0 
```

Comment:
Usually the notation always covers the <= case first. Why this choice?

Reply:
The idea is that we have internal functions who will be shared by both cases. We want to avoid using bridges for this and handle them manually instead.